pkg/proto/datadog/languagedetection/api.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
worth adding a README with a way to generate the golang version of the protobuf
or better, to use `go:generate` and add it to system-probe.build command

Reply:
There should already be one [here](https://github.com/DataDog/datadog-agent/blob/f667ff8b67257438d61cba6c28fae6c5d2b3737b/pkg/proto/datadog/api/README.md#L20). We use `inv generate-protobuf`. I want to be careful with using `go:generate` however. The protobuf version is not standardized, so it may become annoying to build the system probe and have all protobuf files be re-generated with a different proto version.